contracts/rewards/RewardsManager.sol

```diff
@@ -209,28 +202,18 @@ contract RewardsManager is RewardsManagerV3Storage, GraphUpgradeable, IRewardsMa
             return 0;
         }
 
-        // Zero issuance under a rate of 1.0
```

Comment:
Just an observation: we could still keep this check to avoid the CALL to graphToken() and curation() contract if issuance rate is zero. However, as we are deploying it in L2 initially, it will be cheaper and don't have a big effect.

Reply:
Good point, adding it back